custom/icds_reports/templates/icds_reports/icds_app/navigation.directive.html

```diff
@@ -131,6 +131,12 @@
                 </ul>
             </li>
             <hr>
+            <li ng-class="{'active': $location.path().indexOf('/service_delivery_dashboard') !== -1}" ng-if="haveAccessToFeatures">
```

Comment:
This hiding is tested locally?

Reply:
Yes, I have retested it, it is based on `ICDS: Enable access to the features in the ICDS Dashboard reports` flag.